src/core/selfcare_be.tf

```
@@ -0,0 +1,153 @@
+module "selfcare_be" {
+  source = "git::https://github.com/pagopa/azurerm.git//app_service?ref=v2.0.2"
+
+  name = "${var.location}-${var.env_short}"
+
+  ftps_state = "AllAllowed"
+
+  plan_name     = "${var.location}-plan"
+  plan_sku_size = var.app_service_plan_sku
+  plan_sku_tier = var.app_service_plan_sku_tier
+  plan_kind     = var.app_service_plan_kind
```

```suggestion
  plan_kind     = "Linux"
  plan_reserved = true
```
use only linux plans for cost savings.
when we use linux we need also to set

`plan_reserved = true # Mandatory for Linux plan`

ref. https://github.com/pagopa/pagopa-infra/blob/main/src/mock_ec.tf#L28